src/coord/src/catalog/builtin.rs

```diff
@@ -1024,27 +1024,35 @@ pub const PG_TYPE: BuiltinView = BuiltinView {
     mz_types.oid,
     mz_types.name AS typname,
     mz_schemas.oid AS typnamespace,
-    CASE
-        WHEN EXISTS (SELECT 1 FROM mz_catalog.mz_array_types WHERE type_id = mz_types.id) THEN 'a'
-        WHEN EXISTS (SELECT 1 FROM mz_catalog.mz_base_types WHERE type_id = mz_types.id) THEN 'b'
-        WHEN EXISTS (SELECT 1 FROM mz_catalog.mz_map_types WHERE type_id = mz_types.id) THEN 'm'
-    END AS typtype,
+    typtype,
     0::pg_catalog.oid AS typrelid,
     NULL::pg_catalog.oid AS typelem,
-    coalesce(
+    COALESCE(
```

Comment:
Don't bother going back through CI for this, but per PostgreSQL style (https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#sql-style) this is supposed to be lowercase!

Reply:
face palm––a bug in sqlfum.pt

Reply:
@mjibson in case you're interested!

Reply:
Interesting! Maybe one day...